src/content/ai/create-with-ai.md

```diff
@@ -78,18 +97,86 @@ and fix errors, hot reload, get the selected widget, and more. This bridges
 the gap between the AI's natural language understanding, and
 Dart and Flutter's suite of developer tools.
 
-To get started, check out the official documentation for the [Dart and Flutter MCP server][dart-mcp-dart-docs]
+To get started, check out the official documentation for the
+[Dart and Flutter MCP server][dart-mcp-dart-docs]
 on dart.dev and the [Dart and Flutter MCP repository][dart-mcp-github].
 
----
-
-[Gemini Code Assist]: https://codeassist.google/
-[Gemini CLI]: {{site.github}}/google-gemini/gemini-cli
-[Gemini CLI codelab]: https://codelabs.developers.google.com/gemini-cli-hands-on
-[Firebase AI Logic]: {{site.firebase}}/docs/ai-logic
-[firebase-ai-logic-docs]: {{site.firebase}}/docs/ai-logic/get-started
-[How to build agentic apps with Flutter and Firebase AI Logic]: {{site.yt.watch}}/watch?v=xo271p-Fl_4
-[Flutter AI Toolkit]: {{site.url}}/ai-toolkit
 [dart-mcp-dart-docs]: {{site.dart-site}}/tools/mcp-server
 [dart-mcp-github]: {{site.github}}/dart-lang/ai/tree/main/pkgs/dart_mcp_server
 [dart-mcp-flutter-docs]: #dart-and-flutter-mcp-server
+
+## Rules
+
+Rules are a key part of the development experience in many
+AI-driven editors. The rules provide instructions to an
+underlying LLM model, which powers the
+AI assistance features. Rules can be used to do many things,
+including:
+
+*   Customizing AI behavior
+*   Enforcing best practices
+*   Providing project context
+
+### Editors with rules files
+
+Several editors, such as Firebase Studio, have rules files
+that are useful for providing critical context to LLMs.
+
+| Environment/IDE | Rules File | Installation Instructions |
+| :--- | :--- | :--- |
+| Firebase Studio | `airules.md` | [Configure airules.md][] |
+| Copilot powered IDEs | `copilot-instructions.md` | [Configure .github/copilot-instructions.md][] |
+| Cursor | `cursor.md` | [Configure cursorrules.md][] |
+| JetBrains IDEs | `guidelines.md` | [Configure guidelines.md][] |
+| VS Code | `.instructions.md` | [Configure .instructions.md][] |
+| Windsurf | `guidelines.md` | [Configure guidelines.md][] |
+
+[Configure airules.md]: https://firebase.google.com/docs/studio/set-up-gemini#custom-instructions
+[Configure .github/copilot-instructions.md]: https://code.visualstudio.com/docs/copilot/copilot-customization#_custom-instructions
+[Configure cursorrules.md]: https://docs.cursor.com/en/context/rules
+[Configure guidelines.md]: https://www.jetbrains.com/help/junie/customize-guidelines.html
+[Configure .instructions.md]: https://code.visualstudio.com/docs/copilot/copilot-customization#_custom-instructions
+[Configure guidelines.md]: https://docs.windsurf.com/windsurf/cascade/memories#rules
+
+### Convert rules between editors
```

Comment:
I don't think we need to provide instructions on converting between editors. This is a detail that feels better fit for IDE and Rules project docs than Flutter's.

Reply:
This is something I definitely want to keep, but I'll make a change so that people don't have to go to Firebase Studio to grab some rules to work with.